browsers-internal/node18.16.0-chrome116-ff115-edge/README.md

```diff
@@ -0,0 +1,18 @@
+# cypress/browsers-internal:node18.16.0-chrome116-ff115
+
+A complete image with all operating system dependencies for Cypress, and Chrome 116.0.5845.187-1, Firefox 115.0.1, Edge undefined browsers.
```

Comment:
what version of edge is installed? just latest right?

Reply:
yep latest